Dockerfile

```diff
@@ -7,7 +7,7 @@ COPY . .
 RUN go build
 
 FROM restic/restic:0.17.3
-RUN apk add --no-cache rclone bash curl docker-cli
+RUN apk add --no-cache rclone bash curl docker-cli tini
 COPY --from=builder /app/autorestic /usr/bin/autorestic
-ENTRYPOINT []
+ENTRYPOINT [ "/sbin/tini", "--" ]
```

Comment:
There is an old changelog entry about handing signals and graceful shutdown - if I understand `tini` correctly, it'll kill the process on SIGTERM instead of letting it handle its own shutdown

Reply:
”All Tini does is spawn a single child (Tini is meant to be run in a container), and wait for it to exit all the while reaping zombies and performing signal forwarding.“

ref https://github.com/krallin/tini


tinit avoids zombies.

Reply:
> There is an old changelog entry about handing signals and graceful shutdown - if I understand `tini` correctly, it'll kill the process on SIGTERM instead of letting it handle its own shutdown 

https://github.com/restic/restic/blob/master/internal/backend/rclone/backend.go#L85

go cmd no wait caused zombies. this pr is a workaroumd. It works in my homelab

Reply:
> There is an old changelog entry about handing signals and graceful shutdown - if I understand `tini` correctly, it'll kill the process on SIGTERM instead of letting it handle its own shutdown 

https://github.com/restic/restic/blob/master/internal/backend/rclone/backend.go#L85

cmd.Start no wait caused zombies. this pr is a workaroumd. It works in my homelab